ArkGameExample/games/FlappyBird/FlappyBirdEntityCreator.swift

```diff
@@ -30,7 +30,8 @@ enum FlappyBirdEntityCreator {
                              impulse: impulseValue,
                              categoryBitMask: FlappyBirdPhysicsCategory.character,
                              collisionBitMask: FlappyBirdPhysicsCategory.wall | FlappyBirdPhysicsCategory.ceiling,
```

Comment:
Let's use contactTestBitMask only, against just the `.wall`. Anw we only register the collisionHandler between .wall and .character